src/views/Dashboard.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+    <h1>Dashboard</h1>
+    <button @click="logout">Log Out</button>
+</template>
+
+<script lang="ts">
+import router from '../router'
+export default {
+    methods: {
+        logout() {
+            localStorage.removeItem('ID_TOKEN_KEY')
```

Comment:
Add async await here just to make sure homepage is reached only actually after token is removed. Also, move to login page instead.